addons/lookatrandom/addon.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "Point towards random direction",
+  "description": "Adds a \"random position\" option to the \"point towards ()\" block that makes the sprite point in a random direction. This will work for everyone using the project.",
+  "userscripts": [
+    {
+      "url": "code.js",
+      "matches": ["projects"]
+    }
+  ],
+  "versionAdded": "1.29.0",
+  "tags": ["editor"],
+  "enabledByDefault": false,
+  "dynamicEnable": true,
+  "dynamicDisable": true,
+  "credits": ["pufferfish101007#2024: wrote the code", "me (winwinlmfao#8430): made the addon"]
```

```suggestion
  "credits": [
    {
      "name": "pufferfish101007",
      "link": "https://scratch.mit.edu/users/pufferfish101007/"
    },
    {
      "name": "winwinlmfao"
    }
  ]
```
This way the credits will actually display.

And yes, I tested it. It should work.

---

Remember to mark this as resolved :)